web/doppelkopfSchreiben.html

```diff
@@ -21,6 +21,59 @@
       <th class="game">Spieler 2</th>
     </tr>
     </table>
+    <form class="write-blog" id="write-blog" method="post">
+      <div class="write-div">
+        <select id="winner1" class="write-input">
+        </select>
+
+        <select id="winner2" class="write-input">
+        </select>
+      </div>
+
+      <div class="write-div">
+        <div class="write-input">
+          <input type="radio" name="team" id="re" value="re" checked/>
+          <label for="re">Re-Partei</label>
+        </div>
+
+        <div class="write-input">
+          <input type="radio" name="team" id="kontra" value="kontra"/>
+          <label for="kontra">Kontra-Partei</label>
+        </div>
+      </div>
+
+      <div class="write-div">
+        <div class="write-input">
+          <input type="checkbox" id="re-bid" value="reBid"/>
+          <label for="reBid">Re angesagt</label>
+        </div>
+
+        <div class="write-input">
+          <input type="checkbox" id="kontra-bid" value="kontra-bid"/>
```

Comment:
`kontra-bid` und `re-bid` sind momentan nicht miteinander verknüpft, also haben die semantisch keine Zugehörigkeit. Üblicherweise unterteilt man Checkboxes genau wie Radio-Buttons in Gruppen, indem man ihnen dasselbe `name` Attribut zuweist (sowie `name="team"` bei den Radio-Buttons oben).